src/externalized/internals/enums.h

```diff
@@ -1,15 +1,53 @@
 #pragma once
 
+#include "Animation_Data.h"
 #include "game/TacticalAI/NPC.h"
+#include "MapScreen.h"
+#include "MercProfileInfo.h"
+#include "Overhead_Types.h"
+#include "Soldier_Control.h"
+#include "Soldier_Profile_Type.h"
 
 namespace Internals
 {
 	/** Return approach name or NULL if not found. */
 	const char* getApproachName(enum Approach approach);
+	const Approach getApproachEnumFromString(const ST::string& s);
```

Comment:
Please have a look at the Github runner logs for this PR. Marking the return value as `const`  doesn't do anything useful here but adds several new warnings (ignored-qualifiers) to the build.

Reply:
Got it. My next PR is a cleanup one targeting enums.h among other things.